package.json

```
@@ -92,7 +92,7 @@
     "semantic-release": "19.0.3",
     "ts-jest": "29.2.6",
     "ts-node": "10.9.1",
-    "typescript": "4.9.5"
+    "typescript": "^4.9.5"
```

```suggestion
    "typescript": "4.9.5"
```
And run the CLI command `npm i` to pin the dependency in `package-lock.json`.